web/packages/teleterm/src/ui/ConnectMyComputer/index.ts

```diff
@@ -14,4 +14,5 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-export * from './DocumentConnectMyComputerSetup';
+export * from 'teleterm/ui/ConnectMyComputer/DocumentConnectMyComputerSetup/DocumentConnectMyComputerSetup';
```

Comment:
do we want an index file for `DocumentConnectMyComputer`? i thought that was the convention

Reply:
I think it makes sense to add index files only to top-level directories (like `ConnectMyComputer`). Other parts of code will import from this dir, not from `ConnectMyComputer/DocumentConnectMyComputer`. Adding an index file to every directory creates too much noise IMO.

BTW, now I noticed that I used an absolute export in `index.ts`, while it should be relative (probably my IDE did it), thanks :)